src/test/java/core/basesyntax/service/CsvReportGeneratorImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.storage.Storage.storageOfFruits;
+import static junit.framework.TestCase.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+
+class CsvReportGeneratorImplTest {
```

Comment:
add a negative test, what if path is wrong or storage is empty

Reply:
I changed code, because in this method we do not need to have a path